packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -238,10 +238,9 @@
 		<tera-drilldown
 			v-if="dialogIsOpened && currentActiveNode"
 			@on-close-clicked="dialogIsOpened = false"
+			:title="currentActiveNode.displayName"
+			:tooltip="'A brief description of the operator.'"
 		>
-			<template #header>
-				<h5>{{ currentActiveNode.displayName }}</h5>
-			</template>
```

Comment:
I would have seen something more like
```html
<tera-drilldown v-if="dialogIsOpened && currentActiveNode" @on-close-clicked="dialogIsOpened = false">
  <tera-drilldown-header :tooltip="currentActiveNode.tooltip">
    { currentActiveNode.displayName }}
  </tera-drilldown-header>
  ...
</tera-drilldown>
```

Reply:
I put the `tera-drilldown-header` directly in the `tera-drilldown` since I figured that all drilldowns would all have the same headers, do we forsee cases where dont need this header in the drilldown at all?

Reply:
I don't know but this would simplify the use of props, when we can just have it there.
For example we could remove the prop drilling about closing the drilldown:
```vue
<tera-drilldown v-if="dialogIsOpened && currentActiveNode">
  <tera-drilldown-header :tooltip="currentActiveNode.tooltip" @close="dialogIsOpened = false">
    {{ currentActiveNode.displayName }}
  </tera-drilldown-header>
  ...
</tera-drilldown>


```

Reply:
I get that, I guess something else to note is the tab functionality in the header.  The communication for opening/closing tabs is pretty tightly linked between the drilldown and drilldown-header based on the slots we use in the drilldown.  If we were to separate them out like this we would need to handle the tab behaviour on each drilldown we make rather than once inside the drilldown component itself.    

I figure we can keep it like this for now...Ill keep this in mind while I design the main container since I'm a little iffy on using the slots right now to determine the tabs and might end up rewriting the tab behaviour when I work on that PR